02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -515,37 +160,37 @@ To help with consistency, copy the below questions and paste them into your comm
 
 ```
 
+## Everyone: What Should Our Documentation Outcomes Be?
+
+Our purpose is clear: improve the documentation situation
+
+The above summary shows that we should abide by these principles no matter what we do, nor how we do it:
+- **Show, don't tell**: prefer heavily-commented clone-compile-and-play code examples over written explanations
+- **Start unofficial documentation efforts**: let's not slow down core contributors by distracting them with things relating to documentation (e.g. PRs).
+- **Don't recreate the wheel**: look into helping existing efforts before you consider starting your own
+-  **Adhere to Good Docs Guidelines**: follow the above "documentation type" guidelines and the other thoughts expressed in the 'Good Documentation.md' file
+
+The outcomes we are striving to reach have yet to be defined.
+
+We'd like your help. **To prevent "group think," please don't look at our ideas until you have posted yours.**
+
+Please share your idea for _what_ the community should achieve (the reality that is true once finished) **before** sharing your idea for _how_ to achieve it (the step-by-step plan for how to get there).
+
+| Example Type | Comment | Explanation
+| - | - | - |
+| Bad | We should send documentation PRs to specific libraries | Answers 'how to do something' not 'the reality that will be true when we are finished.'
+| Good | **Goal:** The top 30 dependencies used in PS' ecosystem have examples and counterexamples in all of their Pursuit docs.<br>**How:** We make a version of Justin Woo's "ACME" Spago project ([project](https://github.com/justinwoo/acme-spago) & [resulting docs](https://jusrin.dev/acme-spago/)). For packages that are lacking docs or are maintained by core contributors who won't respond quickly, we could use Spago to override those packages with a version that includes more documentation. Then, via `spago docs`, would could create a local copy that has the updated docs. | States a goal but does not determine how that could be achieved. But a few ideas quicly come to mind for how.
```

Comment:
While doing this PR, I think there should be a different example here. I fear people will click on the link to the ACME project and it will short-circuit their thought process, which (after reading this doc) is probably in its best mindset to contribute towards the doc improvement purposes.

However, I'm not sure what that example should be.